lib/command.js

```diff
@@ -755,10 +766,11 @@ Expecting one of '${allowedValues.join("', '")}'`);
    *
    * @param {string} key
    * @param {Object} value
+   * @param {string} [source]
    * @return {Command} `this` command for chaining
    */
 
-  setOptionValue(key, value) {
+  setOptionValue(key, value, source) {
```

Comment:
`source` does not appear to be necessary.

Reply:
Oops, left behind when I moved the source support to private for now. Well spotted.

Reply:
Removed. Thanks.